operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1194,14 +1194,23 @@ spec:
               strimzi.io/kind: cluster-operator
           template:
             metadata:
+              annotations:
+                operator-image: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0-0
+                kafka-images: |
+                  3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0-0
+                  3.8.0=registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0-0
+                kafka-previous-image: registry.redhat.io/amq-streams/kafka-36-rhel9:2.8.0-0
+                kafka-current-image: registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0-0
```

Comment:
The repetition of the Kafka images is strange I presume we can't collapse them or use a variables?

Relatedly we are mixing versions here, in `kafka-images` we have 3.7.0 and 3.8.0 but in previous and current we have `kafka-36` and `kafka-37`. Given the `2.8.0-0` version I think `kafka-images` is right...